app/build.gradle

```diff
@@ -210,35 +210,6 @@ android.applicationVariants.all { variant ->
     println("Build type: " + buildType + " (versionCode = " + variant.mergedFlavor.versionCode + ")")
 }
 
-// -------------------------------------------------------------------------------------------------
-// Generate blocklists
-// -------------------------------------------------------------------------------------------------
-
-def blockListOutputDir = 'src/amazonWebview/res/raw'
-
-task buildBlocklists(type:Copy) {
-    from('../shavar-prod-lists') {
-        include '*.json'
-    }
-    into blockListOutputDir
-
-    // Android can't handle dashes in the filename, so we need to rename:
-    rename 'disconnect-blacklist.json', 'blocklist.json'
-    rename 'disconnect-entitylist.json', 'entitylist.json'
-    // google_mapping.json already has an expected name
-}
-
-clean.doLast {
-    file(blockListOutputDir).deleteDir()
```

Comment:
It looks like this code adds its own action to our cleans, so I'm not sure the verification steps listed in the commit message are enough.  This is a bit of a pain, but I think ideally we would clone this into a new directory, build from there, and re-verify.  If not that, at least rm the directory manually and check again.

Reply:
I additionally did the suggested verification steps. I don't think it's worth the CI/mental load to append this to the commit message though so I'll just leave this conversation unresolved so it's visible to future travellers.